hazelcast-client/src/main/java/com/hazelcast/client/util/ClientAddressCancellableDelegatingFuture.java

```diff
@@ -51,7 +51,7 @@ public ClientAddressCancellableDelegatingFuture(ClientInvocationFuture future,
 
     @Override
     public boolean cancel(boolean mayInterruptIfRunning) {
```

Comment:
Why not pass through to the underlying future and not adding any logic to this method?